tools/wpt/browser.py

```diff
@@ -289,7 +289,7 @@ def find_webdriver(self, channel=None):
         return find_executable("geckodriver")
 
     def get_version_and_channel(self, binary):
-        version_string = call(binary, "--version").strip()
+        version_string = call(binary, "--version").decode('utf8').strip()
```

Comment:
I'm nervous about having to do this at every callsite, vs just making `call` always return text. That's a bigger change, but is there any reason to prefer this approach?

Reply:
No, I don't have any preference. I guess I was a bit over-cautious. 
Just updated the code. Please review. Thank you!